lib/components/app/responsive-webapp.js

```diff
@@ -209,12 +207,12 @@ class RouterWrapperWithAuth0 extends Component {
       showAccessTokenError,
       showLoginError
     } = this.props
-
+    console.log(this.props)
```

Comment:
Remove console statement.

Reply:
Fixed in bbb6987f3a529c8851d6d7e868fabade4632872d.